v20.1/build-a-spring-app-with-cockroachdb-jpa.md

```diff
@@ -655,7 +655,7 @@ We don't go into much detail about hypermedia representation in this tutorial. F
 
 To abstract the database layer, Spring applications use the [`Repository` interface](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories), or some subinterface of `Repository`. This interface maps to a database object, like a table, and its methods map to queries against that object, like a [`SELECT`](selection-queries.html) or an [`INSERT`](insert.html) statement against a table.
 
-[`AccountRepository.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/AccountRepository.java) defines the main repository for the `accounts` table:
+[`AccountRepository.java`](https://github.com/cockroachlabs/roach-data/blob/master/roach-data-jpa/src/main/java/io/roach/data/jpa/AccountRepository.java) defines the main repository for the `accounts` table:
```

Comment:
The url is now actually

https://github.com/cockroachlabs/roach-data/tree/master/roach-data-jpa/src/main/java/io/roach/data/json/AccountRepository.java